spec/factories/places.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :place do
+    name { Faker::Lorem.word }
+    address { Faker::Lorem.word }
+    description { Faker::Lorem.word }
+    average_score { Faker::Number.number(1) }
```

Comment:
you are working very hard to return a random 1 here.
suggestion:
1. random from a valid range
2. return a fixed number, like in line9 . (i dont see any reason to user Faker for a static value)